Modules/IO/ImageBase/include/itkIOTestHelper.h

```diff
@@ -69,13 +75,18 @@ class IOTestHelper
 
   template <typename ImageType, typename ImageIOType>
   static void
-  WriteImage(typename ImageType::Pointer & image, const std::string & filename)
+  WriteImage(typename ImageType::Pointer & image,
+             const std::string &           filename,
+             typename ImageIOType::Pointer imageio = nullptr)
   {
 
     using WriterType = itk::ImageFileWriter<ImageType>;
     typename WriterType::Pointer writer = WriterType::New();
 
-    typename ImageIOType::Pointer imageio(ImageIOType::New());
+    if (imageio.IsNull())
```

Comment:
Is there a difference between doing `if (imageio.IsNull())` vs line 42?

Reply:
I believe `if (imageio)` is equivalent to `if (imageio.IsNotNull())`, so opposite to `if (imageio.IsNull())`.